.github/workflows/npm-publish.yml

```
@@ -0,0 +1,53 @@
+          pnpm -v
+          pnpm install
+      - name: Run test
+        run: |
+          pnpm run test
+      - name: Run build
+        run: |
+          pnpm run build
+      # 发布NPM包
+      - name: 发布NPM包
+        # 执行发布代码
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:49: Double quote to prevent globbing and word splitting

(shellcheck)